src/utils/utils.jl

```diff
@@ -102,3 +96,12 @@ function serialize_case_parameters(case_args::Dict{Symbol, <:Any})
         end
     end
 end
+
+function _get_system_directories(path::String)
+    return (
+        joinpath(path, x) for
+        x in readdir(path) if isdir(joinpath(path, x)) && _is_system_hash_name(x)
+    )
+end
+
+_is_system_hash_name(name::String) = isempty(filter(!isxdigit, name))
```

Comment:
This is now a dumb check. The old hash function produced only numbers. I need to make the new function produce only numbers...or just delete all directories.

Reply:
You could put a consistent prefix on the directory name before the hash and check for that